### ISSUE TITLE
Small correction(s) in documentation

### ISSUE THREAD

**Issue by ggnkua** [zerkman/zest#1]
https://github.com/zerkman/zest/blob/e048e3591176faa2ce5af4f4580a64b266dcc492/doc/build.md?plain=1#L162

Suggested change: ```In Target Architecture, choose i386->ARM (little endian).```

**Comment by ggnkua** [comment]
https://github.com/zerkman/zest/blob/e048e3591176faa2ce5af4f4580a64b266dcc492/doc/build.md?plain=1#L185-L187

```git clone -b xilinx-v2020.2 https://github.com/Xilinx/u-boot-xlnx.git``` is a bit shorter

**Comment by zerkman** [comment]
not sure i386->ARM is clearer than just asking to choose ARM.

also not sure clone -b is easier to understand than a basic clone then a basic checkout.

Not feeling like updating the doc, unless popular demand really demands it :)

**Comment by ggnkua** [comment]
> not sure i386->ARM is clearer than just asking to choose ARM.

This is what the user sees when he writes ```make menuconfig``` for the first time:

![image](https://user-images.githubusercontent.com/5121492/158799548-a01665d3-b7c1-4734-ab2d-b374f18feb37.png)

And this is what the user sees the second time:

![image](https://user-images.githubusercontent.com/5121492/158799669-b9f2dd51-0143-4324-bbc0-c60552d6ce53.png)

So really IMO this is highly confusing for someone who wants to set up the tools etc :).

> also not sure clone -b is easier to understand than a basic clone then a basic checkout.

Basically for me: less commands=less clutter. Ideally most of the commands into the doc could be made into a script. But of course feel free to ignore!

**Comment by zerkman** [comment]
I will leave it as is, sorry.
